.github/workflows/build.yml

```diff
@@ -159,6 +240,14 @@ jobs:
         with:
           name: lifecycle-linux-arm64-sha256
           path: out/lifecycle-v*+linux.arm64.tgz.sha256
+      - uses: actions/upload-artifact@v2
+        with:
+          name: lifecycle-linux-s390x
+          path: out/lifecycle-v*+linux.s390x.tgz
+      - uses: actions/upload-artifact@v2
+        with:
+          name: lifecycle-linux-s390x-sha256
+          path: out/lifecycle-v*+linux.s390x.tgz.sha256
```

Comment:
I think we'll also want to update the release workflow here: https://github.com/buildpacks/lifecycle/blob/aeb177ca468cc814ab47fac0cb878497495b5819/.github/workflows/draft-release.yml#L82

It would be helpful if we could somehow test this on a fork to ensure there aren't any other gotchas that we're failing to find. https://github.com/buildpacks/lifecycle/blob/main/tools/test-fork.sh could be helpful here (though it's a bit finicky, and may require some updating).

Reply:
yes. I updated it to if (urlList.length != 12) { 
this works. With new changes pushed I need to verify this again.